packages/react-api/src/types.d.ts

```diff
@@ -43,6 +43,10 @@ export type SourceProps = {
   type: MapTypesType['QUERY'] | MapTypesType['TABLE'] | MapTypesType['TILESET'];
   connection: string;
   geoColumn?: string;
+  dataResoultion?: number;
+  spatialDataType?: string;
+  spatialDataColumn?: string;
+  spatialFiltersResolution?: number;
```

Comment:
The `spatialFilter` parameter is singular, not plural – assuming these are related, should this be `spatialFilterResolution` instead? Note that this may become a public-facing API once added to `@carto/api-client`, though I do see the note in the AD that we may not expose it.

Reply:
`spatialFiltersResolution` is term already used in platform maps API. Didn't invent this name

I think, creating another slightly different symbol would be confusing, so i would leave it as it is.

@thedae @josemaria-vilaplana I leave this comment to You. Maybe we should broaden reviews of new platform API that is about to be visible in many places us for long time

Reply:
Sorry, platform APIs already spell this as `spatialFilters` so naming problem is only visible in "carto4react" source API.

@donmccurdy I think we're free to fix it in new frontend packages/APIs. I would leave it as it is in c4r which - i believe - is in deprecated mode and we don't refactor things here anymore

Reply:
Ok — sounds good to me!